client/gutenberg/extensions/tiled-gallery/filter-toolbar.js

```diff
@@ -0,0 +1,139 @@
+			</SVG>
+		),
+		title: _x( 'Gingham', 'image style' ),
+		value: 'gingham',
+	},
+];
+
+export default function FilterToolbar( { value, onChange } ) {
+	return (
+		<Dropdown
+			position="bottom right"
+			className="editor-block-switcher"
+			contentClassName="editor-block-switcher__popover"
+			renderToggle={ ( { onToggle, isOpen } ) => {
+				const openOnArrowDown = event => {
+					if ( ! isOpen && event.keyCode === DOWN ) {
+						event.preventDefault();
+						event.stopPropagation();
+						onToggle();
+					}
+				};
+				const label = __( 'Pick an image filter' );
+
+				return (
+					<Toolbar>
+						<IconButton
+							onClick={ onToggle }
+							aria-haspopup="true"
+							aria-expanded={ isOpen }
+							label={ label }
+							tooltip={ label }
+							onKeyDown={ openOnArrowDown }
+							icon={
+								<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+									<Path fill="none" d="M0 0h24v24H0V0z" />
+									<Path d="M19 10v9H4.98V5h9V3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-9h-2zm-2.94-2.06L17 10l.94-2.06L20 7l-2.06-.94L17 4l-.94 2.06L14 7zM12 8l-1.25 2.75L8 12l2.75 1.25L12 16l1.25-2.75L16 12l-2.75-1.25z" />
+								</SVG>
+							}
+						/>
+					</Toolbar>
+				);
+			} }
+			renderContent={ ( { onClose } ) => {
+				const applyOrUnset = nextValue => () => {
+					onChange( value === nextValue ? undefined : nextValue );
+					onClose();
+				};
+				return (
+					<Toolbar
+						icon={
+							<SVG width="24" height="24" viewBox="0 0 24 24">
+								<Rect fill="#f00" x="0" y="0" width="24" height="24" />
+							</SVG>
+						}
+						label={ __( 'Change Text Alignment' ) }
```

Comment:
>Change Text Alignment

:duck:

Reply:
Whoops. This needs to be changed anyways, I doubt it will be a toolbar at all.